llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -954,7 +954,6 @@ void VPlan::prepareToExecute(Value *TripCountV, Value *VectorTripCountV,
 
   IRBuilder<> Builder(State.CFG.PrevBB->getTerminator());
   // FIXME: Model VF * UF computation completely in VPlan.
-  assert(VFxUF.getNumUsers() && "VFxUF expected to always have users");
```

Comment:
Why dropping this check? If this check is dropped, shall it early exit if VFxUF.getNumUsers() == 0?

Reply:
Shall it early exit then if VFxUF.getNumUsers() == 0?

Reply:
Done, thanks!